Azure/azure-linux/tests/auth_conversation_failed.json

```diff
@@ -16,9 +16,6 @@
     "azure_linux": {
       "message": "pam_unix(sudo:auth): conversation failed"
     },
-    "host": {
```

Comment:
I'm wondering why `host.name` was deleted ?

Reply:
It is due to a previous fix on the format, the field is nested under log.hostname, the tests had not been updated